data-serving/data-service/test/case.test.ts

```diff
@@ -43,7 +43,7 @@ describe('GET', () => {
     });
     it('list should return 200 OK', (done) => {
         request(app)
-            .get('/api/cases')
+            .get('/api/cases/')
```

Comment:
Silly question perhaps: is there something about CORS that requires this? Convention for HTTP APIs is not to apply a terminating slash.

Reply:
no, just that we were not consistent with that between the curator api that I removed and the existing data service (that requires a slash).
Let's keep the consistency for now and remove the trailing slash if we need/feel it's better.